src/main/java/org/folio/circulation/rules/Text2Drools.java

```diff
@@ -102,6 +103,7 @@ public Matcher(int indentation, Set<String> criteriaUsed, int maxCriteriumPriori
   private int indentation = 0;
 
   private String[] policyTypes = {"l", "r", "n", "o", "i"};
+  private PolicyValidator policyValidator = (policyType, policies, token) -> {};
```

Comment:
Can this be final? Why does it need initialising here if it is initialised from a constructor parameter?

Reply:
There are two constructors in Text2Drools. One of them doesn't initialize this function. It will help to prevent NPE

Reply:
What is the constructor that does not take a parameter used for?

Reply:
Could that constructor use the parameterised constructor? Which would mean this property could be made final and only initialised in a single place.

Reply:
Default constructor is used in the method org.folio.circulation.rules.Text2Drools#convert(java.lang.String). This method is called from org.folio.circulation.resources.AbstractCirculationRulesEngineResource#reloadRules. 
There is a possibility that after deploy circulation rules will not be updated (the button "save" will not be hit in rule editor) and Check-out can fall with NPE.

Reply:
Can the default constructor delegate to the parameterised constructor?

Reply:
done